apps/communications/contacts/js/contacts.js

```diff
@@ -297,6 +297,12 @@ var Contacts = (function() {
     header.addEventListener('action', handleCancel);
   };
 
+  var setClassToContactList = function(element, activityName, dataType) {
+    if (activityName == 'pick' && dataType.indexOf('text/vcard') > -1) {
+      element.classList.add('disable-fb-items');
```

Comment:
as I said before I would prefer this is done with a more generic and elegant mechanism that could be extended in the future

```js
const SELECT_MODE_CLASS = {
   'pick' : {
     'text/vcard' : ['disable-fb-items']
   }
}

then the code would be something like

var classList = (SELECT_MODE_CLASS[activityName] || {})[activityType] 
for(var j = 0; j < classList.length; j++) {
  element.classList.add(classList[j]);
} 

```